packages/web/hooks/limit-orders/use-orderbook.ts

```diff
@@ -10,34 +10,12 @@ import { useSwapAsset } from "~/hooks/use-swap";
 import { useStore } from "~/stores";
 import { api } from "~/utils/trpc";
 
-interface Orderbook {
-  baseDenom: string;
-  quoteDenom: string;
-  contractAddress: string;
-}
-
 const USDC_DENOM = process.env.NEXT_PUBLIC_IS_TESTNET
```

Comment:
Aren't these the same minimal denoms?

Reply:
Yea forgot to paste in the testnet denom again, good spot!